packages/widget/src/modals/ConnectedWalletModal/ConnectedWalletModal.tsx

```diff
@@ -69,16 +70,9 @@ const ConnectEco = ({
     assetDenom: sourceAsset?.denom,
     chainId: sourceAsset?.chainID,
   });
+
   const account = useMemo(() => {
-    const _chainID = (() => {
```

Comment:
Tried to make improvement here @codingki , let me know if you think this is problematic